sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -42,6 +38,20 @@
 global_media_settings = media_settings_with_comma_dict['GLOBAL_MEDIA_SETTINGS'].pop('1-32')
 media_settings_with_comma_dict['GLOBAL_MEDIA_SETTINGS']['1-5,6,7-20,21-32'] = global_media_settings
 
+
+class helper_logger:
```

Comment:
do we really require this, can be removed ?

Reply:
We don't need this in test_xcvrd so I've removed it from there. This provides code coverage for y_cable_helper, so I've left it there.

Reply:
i still see it in this file.